app/views/public/api_documentation.html.erb

```diff
@@ -118,8 +120,8 @@
     "skipped_slot_percent":"0.5155",
     "ping_time":"205.703",
     "url":"<%= @api_path %>/validators/testnet/12345j84ecQEKUvQ7gYMKxKwKF6PbYXxXxxXXXXXxXXX.json",
-    "stake_pools_list":["Jito", "Marinade"]
-
+    "stake_pools_list":["Jito", "Marinade"],
+    "is_active": true
```

Comment:
I can't see "is_active" attribute in response, no matter which request params I set.